### DISCUSSION TITLE
Project name with parens causes error

### DISCUSSION THREAD

**Discussion by asarnow** [nextpyp/.github#89] [Q&A] [answered]
There seems to be a quoting issue when the project name has parentheses. So as the doctor said, don't do that.

("Doctor, it hurts when I do this!")

![image](https://github.com/nextpyp/.github/assets/4052606/f56913d3-e618-4f5a-b860-4d0b64f1fb30)

**Comment by abartesaghi** [comment, marked as answer]
This was addressed in version [0.6.4](https://nextpyp.app/files/pyp/0.6.4/docs/index.html).
